src/rpc/misc.cpp

```diff
@@ -64,6 +67,13 @@ class DescribeAddressVisitor : public boost::static_visitor<UniValue>
             obj.push_back(Pair("script", GetTxnOutputType(whichType)));
             obj.push_back(Pair("hex", HexStr(subscript.begin(), subscript.end())));
             UniValue a(UniValue::VARR);
+            if (whichType == TX_WITNESS_V0_KEYHASH || whichType == TX_WITNESS_V0_SCRIPTHASH) {
+                UniValue subobj = boost::apply_visitor(*this, addresses[0]);
+                obj.push_back(Pair("embedded", std::move(subobj)));
```

Comment:
This field needs to be added anywhere it's used in help text for RPC calls.

Reply:
You might want to _copy_ the field to the help text rather than _move_ it, though :p

(Use after move here)

Reply:
Done.